### PR TITLE
Adds RandomVisualizer and updates VisualizerGrid tests

### PR DESCRIPTION
**Pull request by bbengfort**
The VisualizerGrid tests were failing potentially because they were
using RadViz visualizers under the hood. To simplify the tests in the
base class I created a RandomVisualizer in tests.random that simply
plots random points using sklearn.datasets.make_blobs in response to
the Visualizer API (e.g. draws on fit).

Using the RandomVisualizer the VisualizerGrid tests were simplified
and testing the layout became simpler. Also added image close
assertions.

### CODE REVIEW

**Comment by bbengfort** [comment]
@NealHumphrey -- this PR fixes the tests for `VisualizerGrid` by adding the `RandomVisualizer` as I discussed in #176. Going to merge it ASAP to get the tests fixed but wanted to ping you for reference in case you were interested.

File: yellowbrick/base.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from sklearn.base import BaseEstimator
 from .utils import get_model_name, isestimator
 from sklearn.model_selection import cross_val_predict as cvp
-from .exceptions import YellowbrickValueError
+from .exceptions import YellowbrickValueError, YellowbrickTypeError
```
Comment (rebeccabilbro) [inline yellowbrick/base.py]: Nice!

File: tests/random.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+from yellowbrick.style import resolve_colors
+
+from sklearn.datasets import make_blobs
+
+
+##########################################################################
+## Random Visualizer
+##########################################################################
+
+class RandomVisualizer(Visualizer):
```
Comment (rebeccabilbro) [inline tests/random.py]: This will definitely come in handy!

**Comment by NealHumphrey** [comment]
Great, thanks @bbengfort! Sorry just haven't had time to get back to yellowbrick since the last lab meetup.

**Pull request merged**